.vscode/launch.json

```diff
@@ -84,6 +84,12 @@
             "request": "launch",
             "mode": "debug",
             "args": [
+                // main
```

Comment:
Commented out?

Reply:
Yes, it is for testing, pre-specified command line arguments, most common use cases covered. Very useful for debugging, or as examples how anyone could debug CLI with args using vscode.

Reply:
Is it ok?

Reply:
OK